libpromises/crypto.c

```diff
@@ -159,30 +159,48 @@ static void RandomSeed(void)
 static const char priv_passphrase[] = "Cfengine passphrase";
 
 /**
+ * @param[in] priv_key_path path to the private key to use (%NULL to use the default)
+ * @param[in] pub_key_path path to the private key to use (%NULL to use the default)
+ * @param[out] priv_key a place to store the loaded private key (or %NULL to
```

Comment:
what is the significance of % in %NULL? - I'm not too familiar with doxygen.

Reply:
I believe that's how you mark literal values. It definitely works in gtk-doc, but I'm not sure about doxygen. Anyway, my Emacs highlights it so I like it. :)

Reply:
:+1: